nexus/api.go

```diff
@@ -119,13 +119,19 @@ func isMediaTypeOctetStream(contentType string) bool {
 
 // Header is a mapping of string to string.
 // It is used throughout the framework to transmit metadata.
+// The keys should be in lower case form.
 type Header map[string]string
```

Comment:
Shouldn't this be a list to match HTTP?

Reply:
No, we deliberately made Nexus headers take only one value.